### DISCUSSION TITLE
Got access to Copilot Nightly version but now the left menu bar button is gone.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#13934] [Copilot Conversations] [answered]
Hello there. Yesterday I saw that I finally had access to Github Copilot Nightly (in vscode), but today, when I wanted it to explain a code for me, the side button that allowed me to ask Copilot for an explanation is gone. Is it normal? Because yesterday the button was there and worked fine.

**Comment by tmlewin** [comment, marked as answer]
> that I finally had access to Github Copilot Nightly (in vscode), but today, when I wanted it to explain a code for me, the side button that allowed me to ask Copilot for an explanation is gone. Is it normal? Because yesterday the button was there and worked fine.

Are you looking for the github copilot lab? it has been decoupled from the nighhtly extension and exists as its own seperate extension here . https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-labs
